tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  font-family: "Poppins", sans-serif;
+}
+body {
```

Comment:
wrap this in a different div its not a good practise to give these properties to body 
                                                                                                              ~Mezab OP

Reply:
@mmusk faran bhai told  me this so its said by @faran4engg 😁

Reply:
Yes Done. @mmusk